examples/multi-container/ruby/Gemfile

```
@@ -18,3 +18,4 @@ gem 'json'
 gem 'pg'
 gem 'sequel'
 gem 'sinatra'
+gem 'thin'
```

nit newline at end of file
```suggestion
gem 'thin'

```